features/fout-meldingen.feature

```diff
@@ -0,0 +1,136 @@
+        En bevat de response de volgende invalidParams
+        | name   | reason                |
+        | rsin   | geen waarde opgegeven |
+        | fields | geen waarde opgegeven |
+
+Rule: fields parameter bevat geen onbekende kenmerk namen
+
+    Abstract Scenario: Er is één of meerdere onbekende kenmerk namen opgegeven
+        Als '/wozobjecten?rsin=0345100002016017&fields=<fields waarde>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                |
+        | title  | Een of meerdere parameters zijn niet correct.         |
+        | status | 400                                                   |
+        | detail | Parameter 'fields' bevat een (deels) ongeldige waarde |
+        En bevat de response de volgende invalidParams
+        | name   | reason                |
+        | fields | <reason omschrijving> |
+
+        Voorbeelden:
+        | fields waarde              | reason omschrijving                                          |
+        | bestaatniet                | ongeldige waarde: 'bestaatniet' opgegeven                    |
+        | bestaatniet,bestaatookniet | ongeldige waarden: 'bestaatniet', 'bestaatookniet' opgegeven |
+
+Rule: type van parameter waarde is correct
+
+    Scenario: type van waarde van een parameter is niet correct
+        Als '/wozobjecten?postcode=1234AA&huisnummer=A' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                               |
+        | title  | Een of meerdere parameters zijn niet correct.        |
+        | status | 400                                                  |
+        | detail | Parameter 'huisnummer' is niet van het correcte type |
+        En bevat de response de volgende invalidParams
+        | name       | reason                   |
+        | huisnummer | waarde 'A' is geen getal |
+
+Rule: Een zoek actuele woz objecten aanroep mag slechts één identificatie parameter bevatten
+
+    Abstract Scenario: Er zijn meerdere identificatie parameters opgegeven
+        Als '/wozobjecten<query string>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | detail | <fout detail>                                 |
+
+        Voorbeelden:
+        | query string                                        | fout detail                                                                         |
+        | ?rsin=12345&kvkNummer=12345                         | Alleen parameter 'rsin' of 'kvkNummer' moet worden opgegeven                        |
+        | ?adresseerbaarObjectIdentificatie=123456&rsin=12345 | Alleen parameter 'adresseerbaarObjectIdentificatie' of 'rsin' moet worden opgegeven |
+
+Rule: Zoeken met postcode kan alleen in combinatie met huisnummer?
```

Comment:
Ik zie een vraagteken. Weet je deze niet zeker ? 
Kunnen we geen collectie WOZ_objecten van 1 postcode opvragen ?

Reply:
Klopt. Ik kan niet afleiden wat het gewenste gedrag moet zijn. In de goals canvas concludeer ik postcode en huisnummer. Uit de OAS concludeer ik dat je ook kan zoeken met alleen huisnummer

Reply:
het is postcode in combinatie met huisnummer :-)